templates/cinder/config/cinder.conf

```diff
@@ -95,12 +95,3 @@ lock_path = /var/locks/openstack/os-brick
 #volume_backend_name=nfs
 #volume_driver=cinder.volume.drivers.nfs.NfsDriver
 #nfs_shares_config=/etc/cinder/nfs_shares
-
```

Comment:
this and the changes in the README seem unrelated to kuttl tests though. This seems like a separate issue that was fixed (perhaps that you discovered working on the kuttl?).

Reply:
I probably pulled this from somewhere trying to rebase, lemme check

Reply:
I have rebased the PR, this should be gone now